From 58a840d0671091e3064e36244790ef8839da87d6 Mon Sep 17 00:00:00 2001
From: Oleg Pykhalov <go.wigust@gmail.com>
Date: Wed, 12 Aug 2020 17:44:17 +0300
Subject: [PATCH] build: Fix build for Emacs 27

<https://gitlab.com/emacs-guix/emacs-guix/-/merge_requests/5>.

* elisp/local.mk (AUTOLOADS): Add '-l autoload' option to make it
compatible with Emacs 27.
---
 elisp/local.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elisp/local.mk b/elisp/local.mk
index a871aa9..3bc0a27 100644
--- a/elisp/local.mk
+++ b/elisp/local.mk
@@ -109,7 +109,7 @@ nodist_lisp_DATA =				\
   $(AUTOLOADS)
 
 $(AUTOLOADS): $(EL_FILES)
-	$(AM_V_GEN) $(EMACS) -Q --batch --eval					\
+	$(AM_V_GEN) $(EMACS) -Q --batch -l autoload --eval			\
 	  "(let ((backup-inhibited t)						\
 	         (generated-autoload-file \"$(abs_builddir)/$(AUTOLOADS)\"))	\
 	     (update-directory-autoloads \"$(abs_srcdir)/%D%\"))"
-- 
GitLab

